Assets/Plugins/CountlySDK/Persistance/Repositories/Impls/NonViewEventRepository.cs

```diff
@@ -13,7 +13,7 @@ public NonViewEventRepository(Dao<EventEntity> dao, SegmentDao segmentDao, Count
 
         protected override bool ValidateModelBeforeEnqueue(CountlyEventModel model)
         {
-            return !model.Key.Equals(CountlyEventModel.ViewEvent);
+            return true;
```

Comment:
Why this change?

Reply:
this check preventing the event with key '[CLY]_view'  from adding to the event repository.